chapters/en/Unit 2 - Convolutional Neural Networks/convnext.mdx

```diff
@@ -0,0 +1,82 @@
+
+# ConvNext - A ConvNet for the 2020s (2022)
+
+## Introduction
+The paper "A ConvNet for the 2020s" proposed the ConvNext architecture in 2022 by a team of researchers from Facebook AI Research, including Zhuang Liu, Hanzi Mao, Chao-Yuan Wu, Christoph Feichtenhofer, Trevor Darrell, and Saining Xie.
```

Comment:
the paper details can be added to the Reference section

Reply:
changed